packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -138,7 +138,8 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
         buttonProps,
         {
           'onClick': this._onToggleMenu,
-          'aria-expanded': this.state.menuProps ? true : false
+          'aria-expanded': this.state.menuProps ? true : false,
+          'aria-haspopup': true
```

Comment:
`hasPopup` should probably be `!!this.props.menuProps`. If the button doesn't have a menu this attribute shouldn't be true.